analysis/load_program.go

```diff
@@ -42,7 +42,7 @@ const PkgLoadMode = packages.NeedName |
 func LoadProgram(config *packages.Config,
 	platform string,
 	buildmode ssa.BuilderMode,
-	args []string) (*ssa.Program, error) {
+	args []string) (*ssa.Program, []*packages.Package, error) {
```

Comment:
nit: could make this return a `LoadedProgram` struct similar to `LoadedTestProgram`